journalbeat/magefile.go

```diff
@@ -22,20 +22,42 @@ package main
 import (
 	"context"
 	"fmt"
+	"strings"
 	"time"
 
 	"github.com/magefile/mage/mg"
 	"github.com/magefile/mage/sh"
+	"github.com/pkg/errors"
 
 	"github.com/elastic/beats/dev-tools/mage"
 )
 
 func init() {
 	mage.BeatDescription = "Journalbeat ships systemd journal entries to Elasticsearch or Logstash."
 
-	// TODO filter platforms
+	mage.Platforms = mage.Platforms.Filter("linux")
```

Comment:
Given then ppc64 and mips64 can't be build at , I'm wondering if we should filter them like `linux !linux/ppc64 !linux/mips64`. The downside is that it will silently ignore those platforms, but the upside is that you can build the all of beats with `PLATFORMS='+all linux/mips' make snapshot` and it will just work.